infracost-usage-example.yml

```
@@ -964,6 +964,9 @@ resource_usage:
   azurerm_firewall.my_firewall:
     monthly_data_processed_gb: 100000 # Monthly data processed by the firewall in GB.
 
+  azurerm_iothub_dps.my_iothub_dps:
+    monthly_operations: 10000
```

We usually try to put a comment near the usage param value as documentation, so maybe we could include something like this:
```suggestion
    monthly_operations: 10000 # Monthly number of device provisioning operations
```